util/gh-pages/index.html

```diff
@@ -1,10 +1,17 @@
 <!DOCTYPE html>
+<!-- 
+Welcome to a Clippy's lint list, at least the source code of it. If you are
+interested in contributing to this website checkout `util/gh-pages/index.html`
+inside the rust-clippy repository.
+
+Otherwise, have a great day =^.^=
+-->
 <html lang="en">
 <head>
     <meta charset="UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
 
-    <title>ALL the Clippy Lints</title>
+    <title>Clippy's lint list</title>
```

Comment:
Why not just 

```suggestion
    <title>Clippy Lints</title>
```

? Anyway, use title case here. Same below.

Reply:
I would like a different title than "ALL the Clippy Lints" as it always seems a bit long and weird when used in a sentence :sweat_smile:. "Clippy's Lint List" has a nice ring to it, but I'm not sure if it totally fits in the context and tone of Rust.  So, it's your call "Clippy Lints" is also fine with me :)

Reply:
I would prefer just `Clippy Lints`, since the fact that it is a list is pretty obvious. IMO shorter headings are always better, if it has the same meaning.